server/tests/DAL/transac_stat.dao.test.js

```diff
@@ -183,19 +185,20 @@ describe("Test Transac Stat DAO", () => {
                         "FROM ACCOUNTING_CLIENT_STAT ACS ",
                         ", NAME_CONNECTION NC, NAME_QUALITY NQ1, NAME RESP ",
                         ", NAME_QUALITY NQ2 ",
-                        "WHERE ACS.YEAR_MONTH in (?) AND ACS.CONNECTION_ID=3 AND ACS.STAT_TYPE=1",
+                        "WHERE ACS.YEAR_MONTH in (?) AND ACS.STAT_TYPE=1",
```

Comment:
If you decide to change ACS_CONNECTION_ID = 3, you will have to change some line in the tests of course like this one for example

Reply:
Yes check later in the WHERE, it is under NQ2.DROPDOWN_CODE=? and it it replaced in every tests in both test files